osu.Game/Online/Multiplayer/MultiplayerClient.cs

```diff
@@ -783,7 +783,7 @@ private void updateLocalRoomSettings(MultiplayerRoomSettings settings)
             RoomUpdated?.Invoke();
         }
 
-        private PlaylistItem createPlaylistItem(MultiplayerPlaylistItem item) => new PlaylistItem(new APIBeatmap { OnlineID = item.BeatmapID })
+        private PlaylistItem createPlaylistItem(MultiplayerPlaylistItem item) => new PlaylistItem(new APIBeatmap { OnlineID = item.BeatmapID, StarRating = item.StarRating })
```

Comment:
I'm confused by this. The relevant issue thread considers two options: either the client should be sending this star rating value, or the server should be fetching it from db and populating it server-side. So this is the first, but you also opened https://github.com/ppy/osu-server-spectator/pull/169, which is doing... the second?

What is the reasoning here? And if we're doing server-side anyways, then if I'm honest, I'd rather drop the client-side population and be 100% server-authoritative. While this particular piece of data isn't really sensitive in any way, I'd rather prevent clients from being able to send bogus star ratings if possible.